docs/concepts/plugins.md

```diff
@@ -0,0 +1,93 @@
+---
+description: Plugins can add new functionality to Fresh without requiring significant complexity.
+---
+
+> Plugins are only available in Fresh 1.1 and above.
+
+Plugins can dynamically add new functionality to Fresh without exposing
+significant complexity to the user. Users can add plugins by importing and
+initializing them in their `main.ts` file:
+
+```ts
+// main.ts
+
+// ...
+
+import twindPlugin from "$fresh/plugins/twind.ts";
+import twindConfig from "./twind.config.js";
+
+await start(manifest, {
+  plugins: [
+    // This line configures Fresh to use the first-party twind plugin.
+    twindPlugin(twindConfig),
```

Comment:
How are plugins run, in order or at random?

Reply:
Additionally I think it might be a good idea to state that third-party plugins are also allowed as the example above can imply that third-party plugins aren't allowed yet.